lib/plugins/jsonBodyParser.js

```diff
@@ -7,6 +7,9 @@ var errors = require('restify-errors');
 
 var bodyReader = require('./bodyReader');
 
+///---local globals
+var jsonPatternMatcher = new RegExp('^application/[a-zA-Z.]+\\+json');
```

Comment:
It looks like these two lines are the same, perhaps we should break it out into it's own file to avoid them falling out of sync?

I.E. `reqiure('jsonPatternMatcher')` and `module.exports = new RegExp...`?

I can see myself reviewing a PR in the future that modifies this line and not realizing that it exists multiple places in the codebase.

Reply:
Done!